submissions/f3d0t/task_js_dom/script.js

```diff
@@ -37,44 +37,41 @@ const createMenuLinks = (array, container) => {
 		let li = document.createElement("li");
 		li.textContent = element.name;
 		li.dataset.id = element.id;
-		li.className = "";
+		li.classList = "";
 		wrapper.appendChild(li);
 	});
 	container.appendChild(wrapper);
 };
 
-const paintContent = (id, arr) => {
-	let wrapper = document.createElement("main");
-	wrapper.className = "main";
+const getMainContent = (id, arr) => {
+	let wrapper = document.createElement("div");
 	let item = arr.find((item) => item.id == id);
 	let h2 = document.createElement("h2");
-	h2.className = "main__name";
+	h2.classList = "main__name";
```

Comment:
Read how to use [classList](https://developer.mozilla.org/en/docs/Web/API/Element/classList)